base-template.sh

```diff
@@ -421,19 +439,52 @@ update_shared_hooks_if_appropriate() {
 #   1 in case a hook fails, 0 otherwise
 #####################################################
 execute_shared_hooks() {
-    for SHARED_ROOT in ~/.githooks/shared/*; do
-        REMOTE_URL=$(cd "$SHARED_ROOT" && git config --get remote.origin.url)
+    # split on comma and newline
+    IFS=",
+    "
+    for SHARED_REPO in $SHARED_REPOS_LIST; do
+
+        set_shared_root "$SHARED_REPO"
+
+        # Fail if the shared root is not available (if enabled)
+        FAIL_ON_NOT_EXISTING=$(git config --global --get githooks.failOnNotExistingSharedHooks)
```

Comment:
To follow the lowercase dot-separated pattern (like `githooks.trust.all`) can you please rename this?
Something like `githooks.fail.on.missing.shared.hooks` perhaps?

Reply:
That would actuall insert
```ini
[githooks "fail.on.missing.shared"]
	hooks = true
``` 
I dont think you want that :-)

Reply:
May be, `githooks.fail.onNotExistingSharedHooks` would work.

Reply:
OK, let's leave it as is.